# mode: 0->PositionCommand | 1->VelocityCommand
uint8 mode
float64[] values
